master/static/migrations/20210114153111_migrate-trial-id-to-bigint.down.sql

```diff
@@ -0,0 +1,27 @@
+DO $$
+    BEGIN
+        IF EXISTS (SELECT 1
+                  FROM information_schema.columns
+                  WHERE table_schema = 'public'
+                    AND table_name = 'trial_logs'
+                    AND column_name = 'id'
+                    AND numeric_precision = 64) THEN
```

Comment:
IF NOT EXISTS (SELECT 1
				  FROM public.trial_logs
				  WHERE id > 2147483647)
		    AND EXISTS (SELECT 1
                                          FROM information_schema.columns
                                          WHERE table_schema = 'public'
                                            AND table_name = 'trial_logs'
                                            AND column_name = 'id'
                                            AND numeric_precision = 64) THEN